database.go

```diff
@@ -126,11 +126,11 @@ func listUsers() {
 	checkErr(err)
 	defer rows.Close()
 	log.Println("List of current users")
-	var uid, userType int
+	var uid, isRoot int
```

Comment:
This also should be bool, and everywhere else

Reply:
You right, I also change the numerical values to booleans in the addUser call